Gemfile.lock

```diff
@@ -82,80 +82,80 @@ GEM
     crass (1.0.6)
     erubi (1.9.0)
     execjs (2.7.0)
-    ffi (1.12.2)
+    ffi (1.13.1)
     globalid (0.4.2)
       activesupport (>= 4.2.0)
-    i18n (1.8.2)
+    i18n (1.8.5)
       concurrent-ruby (~> 1.0)
     jbuilder (2.10.0)
       activesupport (>= 5.0.0)
-    jquery-rails (4.3.5)
+    jquery-rails (4.4.0)
       rails-dom-testing (>= 1, < 3)
       railties (>= 4.2.0)
       thor (>= 0.14, < 2.0)
     listen (3.1.5)
       rb-fsevent (~> 0.9, >= 0.9.4)
       rb-inotify (~> 0.9, >= 0.9.7)
       ruby_dep (~> 1.2)
-    loofah (2.4.0)
+    loofah (2.6.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
     method_source (1.0.0)
-    mimemagic (0.3.4)
+    mimemagic (0.3.5)
```

Comment:
### [mimemagic](https://github.com/minad/mimemagic)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.3.4 → 0.3.5

[_[change-log](https://github.com/minad/mimemagic/blob/master/CHANGELOG.md), [source-code](https://github.com/minad/mimemagic)_]





<details>
<summary>Commits</summary>

A change of **2** commits. See the full changes on [the compare page](https://github.com/minad/mimemagic/compare/v0.3.4...v0.3.5).

These are the individual commits:
- (51bdc84) [Mimetype extensions are now ordered by freedesktop.org's pri…](https://github.com/minad/mimemagic/commit/51bdc8407811a95d301e6a63dd93b30e7430cb80)
- (01f92d8) [v0.3.5](https://github.com/minad/mimemagic/commit/01f92d86d15d85cfd0f20dabd025dcbd36a8a60f)


</details>